src/transmitter/index.ts

```diff
@@ -122,5 +122,9 @@ function shouldRetryRequest(err: IRequestError, networkErrorMessages: string[],
     return true;
   }
 
+  if (err.code === RequestCodeError.EAI_AGAIN) {
```

Comment:
no need to specify "k8s-monitor" in the commit title; this is the Kubernetes-Monitor repo. unless specified otherwise, everything here is in the context of the kubernetes monitor.
so in this case, ```fix: retry transmitting data on EAI_AGAIN error codes``` might be more fitting